fuse_doc/doc/.templates/full_globaltoc.html

```diff
@@ -5,16 +5,16 @@ <h3><a href="{{ pathto(master_doc) }}">{{ _('Further Documentation') }}</a></h3>
 <ul>
   <li class="toctree-l1">API Documentation</li>
   <ul>
-    <li class="toctree-l1"><a href="../../fuse_constraints/api/index.html"/>fuse_constraints</a></li>
-    <li class="toctree-l1"><a href="../../fuse_core/api/index.html"/>fuse_core</a></li>
-    <li class="toctree-l1"><a href="../../fuse_graphs/api/index.html"/>fuse_graphs</a></li>
-    <li class="toctree-l1"><a href="../../fuse_loss/api/index.html"/>fuse_loss</a></li>
-    <li class="toctree-l1"><a href="../../fuse_models/api/index.html"/>fuse_models</a></li>
-    <li class="toctree-l1"><a href="../../fuse_msgs/html/msg/"/>fuse_msgs</a></li>
-    <li class="toctree-l1"><a href="../../fuse_optimizers/api/index.html"/>fuse_optimizers</a></li>
-    <li class="toctree-l1"><a href="../../fuse_publishers/api/index.html"/>fuse_publishers</a></li>
-    <li class="toctree-l1"><a href="../../fuse_variables/api/index.html"/>fuse_variables</a></li>
-    <li class="toctree-l1"><a href="../../fuse_viz/api/index.html"/>fuse_viz</a></li>
+    <li class="toctree-l1"><a href="../fuse_constraints/index.html"/>fuse_constraints</a></li>
+    <li class="toctree-l1"><a href="../fuse_core/index.html"/>fuse_core</a></li>
+    <li class="toctree-l1"><a href="../fuse_graphs/index.html"/>fuse_graphs</a></li>
+    <li class="toctree-l1"><a href="../fuse_loss/index.html"/>fuse_loss</a></li>
+    <li class="toctree-l1"><a href="../fuse_models/index.html"/>fuse_models</a></li>
+    <li class="toctree-l1"><a href="../fuse_msgs/index.html"/>fuse_msgs</a></li>
```

Comment:
This one is special, and a broken link for now. rosdoc2 doesn't seem to generate any files for interface packages. I left it in so that it will hopefully start working once rosdoc2 supports that.